crates/pg_cli/src/commands/check.rs

```diff
@@ -56,7 +62,7 @@ impl CommandRunner for CheckCommandPayload {
     }
 
     fn should_write(&self) -> bool {
-        self.write || self.fix
+        false
```

Comment:
sanity check: did you just change this for debugging?

Reply:
no, I removed all the `--fix` stuff because ware not fixing anything. its a leftover from biomes `check` command.